ballerina/modules/soap12/soap12.bal

```diff
@@ -45,25 +46,34 @@ public isolated client class Client {
 
     # Sends SOAP request and expects a response.
     # ```ballerina
-    # xml|mime:Entity[] response = check soapClient->sendReceive(body);
+    # xml response = check soapClient->sendReceive(body);
+    #      -- OR --
+    # mime:Entity[] response = check soapClient->sendReceive(body);
     # ```
     #
     # + body - SOAP request body as an `XML` or `mime:Entity[]` to work with SOAP attachments
     # + action - SOAP action as a `string`
     # + headers - SOAP headers as a `map<string|string[]>`
     # + path - The resource path
+    # + typeParam - Default parameter use to infer the user specified type.
     # + return - If successful, returns the response. Else, returns an error
-    remote isolated function sendReceive(xml|mime:Entity[] body, string? action = (),
+    remote isolated function sendReceive(xml|mime:Entity[] body, string? action = (), map<string|string[]> headers = {},
+                                         string path = "", typedesc<any> typeParam = <>)
```

Comment:
here also

Reply:
also shall we use another variable for `typeParam`. I think we used `T` in other modules

Reply:
Fixed by [74145ed](https://github.com/ballerina-platform/module-ballerina-soap/pull/112/commits/74145ed5edd44abb461d9a099bef12d689d49fbb)